Documentation/Links/LinkActions.md

```diff
@@ -51,6 +51,40 @@ When using the `Make this item a link` option, the following settings are availa
 | `Menu item` | same as above. |
 | `Open link in Context Blade` | same as above. |
 
+## ARM Action Settings
+
+Use this setting to invoke an ARM action by specifying the ARM api details.
+
+### ARM Action Settings Tab
+
+This section defines the ARM action API.
+
+| Source | Explanation |
+|:------------- |:-------------|
+|`ARM Action path` | The ARM action path. eg /subscriptions/:subscription/resourceGroups/:resourceGroup/someAction?api-version=:apiversion. Select from `Cell`, `Column`, `Static Value`, or `Parameter` in [Link sources](#link-sources).|
+|`Http Method` | Select an HTTP method. The available choices are `POST`, `PUT`, `PATCH`, `DELETE`|
+|`Long Operation` | Long Operations will poll the URI from the Azure-AsyncOperation response header from the original operation.|

```

Comment:
>Long Operation [](http://example.com/codeflow?start=2&length=14)

should probably be be a link for this too, so people know what this means?  both here and possibly in the infoballoon in the ui?